### ISSUE TITLE
Docs are very hard to follow for acme dns

### ISSUE THREAD

**Issue by chiju** [certbot/certbot#7297]
I can see an example for acme for HTTP challenge <https://raw.githubusercontent.com/certbot/certbot/master/acme/examples/http01_example.py> but it will be great if there is another example for dns challenge, it seems really hard to follow docs for acme

**Comment by ohemorange** [comment]
This is great feedback, thank you!

**Comment by chiju** [comment]
Thanks for the resposne
